src/app/components/NavBar.tsx

```diff
@@ -1,9 +1,11 @@
+"use client";
+
```

Comment:
Can you remove this, not sure if this is causing the site to crash?

Reply:
Tested locally on my device, even when removed, the site crashes.

Reply:
I just removed  the "use client" directive but it s very strange that the error would come from this..

On my end when I visit both both deploy preview URLs below it s working fine
<img width="1402" alt="Screenshot 2024-08-04 at 20 38 02" src="https://github.com/user-attachments/assets/15a3d2e9-4dcd-464d-bf39-815bf2a15d59">